modules/services.sh

```diff
@@ -1,5 +1,15 @@
 function services {
-  check_missing_binary docker-compose "docker-compose is missing\ninstall instructions can be found in\nhttps://github.com/docker/compose"
+  # check_missing_binary docker-compose "docker-compose is missing\ninstall instructions can be found in\nhttps://github.com/docker/compose"
+
+  if docker-compose version 2>/dev/null | grep -q "Docker Compose version"; then
+    dockercompose="docker compose"
+  elif docker-compose version 2>/dev/null | grep -q "docker-compose version"; then
+    dockercompose="docker-compose"
```

Comment:
think we should switch them first
and then use `which` for `docker-compose`
and use ? for `docker compose`

Reply:
```
root@pirateship:~# which docker-compose
/usr/local/bin/docker-compose
root@pirateship:~# docker compose
docker: 'compose' is not a docker command.
See 'docker --help'
```
and
```
root@egdirbmac:~# which docker-compose
root@egdirbmac:~# docker compose

Usage:  docker compose [OPTIONS] COMMAND

Define and run multi-container applications with Docker

Options:
...
```